src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from "@angular/core";
+import { EntityMapperService } from "../../../../core/entity/entity-mapper.service";
+import { Child } from "../../model/child";
+import { DynamicComponent } from "../../../../core/view/dynamic-components/dynamic-component.decorator";
+import { OnInitDynamicComponent } from "../../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+import { MatTableDataSource } from "@angular/material/table";
+import { MatSort } from "@angular/material/sort";
+import { MatPaginator } from "@angular/material/paginator";
+import { ViewChild } from "@angular/core";
+@DynamicComponent("BirthdayDashboard")
+@Component({
+  selector: "app-birthday-dashboard",
+  templateUrl: "./birthday-dashboard.component.html",
+  styleUrls: ["./birthday-dashboard.component.scss"],
+})
+export class BirthdayDashboardComponent implements OnInitDynamicComponent {
+  children: Child[] = [];
+  dataSource: MatTableDataSource<Child>;
+  displayedColumns: string[] = ['entityid', 'Birthday', 'age'];
+  @ViewChild(MatPaginator) paginator: MatPaginator;
+  @ViewChild(MatSort) sort: MatSort;
+  currentUser: Child[];
+  constructor(private entityMapper: EntityMapperService) {
+    // TODO sort the children based on the date of birth
+    this.currentUser = this.children;
+    this.dataSource = new MatTableDataSource(this.children);
```

Comment:
At this point `children` is still an empty array `[]`. So nothing will be shown in the list. The real values (with all the children) are only set (asynchronously) in line 32. See the comment that I left there.